docs/docs/concepts/low_level.md

```diff
@@ -401,7 +401,17 @@ def node_a(state, config):
     ...
 ```
 
-See [this guide](../how-tos/configuration.ipynb) for a full breakdown on configuration
+See [this guide](../how-tos/configuration.ipynb) for a full breakdown on configuration.
+
+### Recursion Limit Configuration
+
+The recursion limit sets the amount of super-steps (read more about what a super-step is [here](#graphs)) your graph can execute before failing. By default this value is set to 25. The recursion limit can be set on any graph, and is passed in the outer `config` dictionary, not inside of the `configurable` key like user defined configurations as in the example below:
```

Comment:
(nit) let's either just link on `[super-steps](...url)` or shorten to something like (read more `[here[(...)`)

consider something like this


The recursion limit sets the maximum number of [super-steps](...url) the graph can execute during a single execution. Once the limit is reached, LangGraph will raise `GraphRecursionError`. By default this value is set to 25 steps. The recursion limit can be set on any graph at runtime, and is passed to `.invoke` / `.stream` via the `config` dictionary. Importantly, `recursion_limit` is a standalone config key and should **not** be passed inside the `configurable` key as all other user-defined configuration. See the example below: